sentry-apollo/src/test/java/io/sentry/apollo/SentryApolloInterceptorTest.kt

```diff
@@ -150,6 +154,16 @@ class SentryApolloInterceptorTest {
         }, anyOrNull())
     }
 
+    @Test
+    fun `adds breadcrumb when http calls succeeds`() {
+        executeQuery()
+        verify(fixture.hub).addBreadcrumb(com.nhaarman.mockitokotlin2.check<Breadcrumb> {
```

Comment:
```suggestion
        verify(fixture.hub).addBreadcrumb(check<Breadcrumb> {
```

we can just import the check extension method